parsing_scripts/account.rb

```diff
@@ -0,0 +1,33 @@
+require 'pry'
+require 'nokogiri'
+require 'open-uri'
+
+class Account
+  
+  def initialize (browser)
+    @browser = browser
+  end 
+
+  def get_account_info
```

Comment:
well, if you really need this method what about

```rb
def get_account_info
    get_account_hash
end
```

Reply:
If this method does the same as another one -> get rid of one of them

Reply:
Fixed